src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -734,4 +734,7 @@
   <data name="SaveSnippetNamePrefix" xml:space="preserve">
     <value>Save Snippet</value>
   </data>
+  <data name="QuickFixCommandKey" xml:space="preserve">
+    <value>Quick fix</value>

```

Comment:
Shouldn't this be "open quick fix menu" or something? All of the other actions are verbs...